### PR TITLE
Blazor Project structure topic UE pass

### LINKED ISSUE #21240: Surprising mention of WeatherForecastService
Hi, 

It looks like this section was lifted from an example or tutorial as it mentions the "WeatherForecastService" which is not introduced (e.g. as "an example service used by the app").

This is confusing, thanks, Hugh

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d31248d7-d2c2-1a17-3173-889a0d9bb6e8
* Version Independent ID: 6f27ff37-6b3a-57f5-353a-5cf5aabba7cf
* Content: [ASP.NET Core Blazor project structure](https://docs.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-5.0)
* Content Source: [aspnetcore/blazor/project-structure.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/project-structure.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### LINKED ISSUE #21241: Consider describing structure separately for Blazor WebAssembly and Blazor Server templates
The structure of a Blazer project itself is already rather complex (on first read). For me it would be easier if the structure + content would be described separately for a WebAssembly template and a Server template. The "inlining" now is hard to parse

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d31248d7-d2c2-1a17-3173-889a0d9bb6e8
* Version Independent ID: 6f27ff37-6b3a-57f5-353a-5cf5aabba7cf
* Content: [ASP.NET Core Blazor project structure](https://docs.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-5.0)
* Content Source: [aspnetcore/blazor/project-structure.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/project-structure.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #19286
Fixes #21241
Fixes #21240

Thanks @hugh-maaskant! :rocket:

### CODE REVIEW

**Comment by hugh-maaskant** [comment]
Hi Luke,

Thanks - looks a lot more accessible now. 

Still it would have helped me if in the beginning of the section there was a sentence to introduce the WeaterForecastService, the Counter and the FetchData component. These are very useful to demonstrate the concepts, but I did not expect them in a template. Note that on first reading I had not yet created a Blazer project, so I had not seen them yet.. Maybe something along the lines of:

The Blazer templates for Blazor Server and Blazor WebWebAssemly create the initial files and directory structure for a Blazor solution. Furthermore they are populated with demonstration code for including a component (FetchData), updating data based on user interaction (Counter) as well as adding and using services (WeatherForecastService).

That way the use of these names would make more sense.

I leave it up to you to decide if this is worthwhile, though.

Best Regards,

Hugh

——
Hugh Maaskant
The Netherlands

**Comment by guardrex** [comment]
Yeah ... I can shoot that in there.
